### ISSUE TITLE
Some module command docs still use hardcoded . command prefix

### ISSUE THREAD

**Issue by dgw** [sopel-irc/sopel#1019]
Great progress was made toward fixing up the dynamic help with the `core.help_prefix` option and fix in bf6f0ff, but there are still some modules that show a hardcoded `.command` syntax in the middle of the example. I noticed this in meetbot earlier:

``` IRC
[00:43:14] <~dgw> ;help startmeeting
[00:43:15] <&Kaede> dgw:     Start a meeting.
[00:43:16] <&Kaede> dgw:     https://github.com/sopel-irc/sopel/wiki/Using-the-meetbot-module
[00:43:17] <&Kaede> dgw: e.g. ;startmeeting title or .startmeeting
```

Is it desirable to try and pick those up when preparing the docs for `help.py`'s use (e.g. by storing the first `<char>` of the string passed to `@example` and then replacing other matches of `<char>word`? _(Complicated, possibly error-prone.)_

Is it more desirable to just fix modules' `@example` calls to only have one usage per decorator? _(Simple and establishes a convention that makes the current code work perfectly if followed.)_

Or is it most desirable to just accept that there's probably no "perfect" way to handle non-default `prefix`es?

I'm willing to take a shot at implementing either of the options that require code changes, just wondering which direction is better in the community's collective opinion—or if there's another option I hadn't thought of.

**Comment by miigotu** [comment]
This also happens with custom modules.

```
<miigotu> !help issues
<SickRage> miigotu: Replies with issue tracker or pr links
<SickRage> miigotu: !pr, !pull, !pulls, !issue, !issues (new|#3|3|)
<SickRage> miigotu: e.g. .pull #111
<miigotu> !issues
<SickRage> Issues: https://github.com/SickRage/sickrage-issues/issues
```

**Comment by Exirel** [comment]
With #1403 merged I think this bug should be easier to fix. A thorough review of `sopel.modules.*` is requested!

**Comment by dgw** [comment]
I note that most of these were already fixed in #1403 itself, but a couple stragglers were still kicking about. PR momentarily.
